libvast/vast/system/local_segment_store.hpp

```diff
@@ -12,6 +12,7 @@
 
 #include "vast/chunk.hpp"
 #include "vast/segment_builder.hpp"
+#include "vast/system/accountant.hpp"
```

Comment:
This include was added erroneously by clangd. The same in a couple of other files.